internal/util/util.go

```diff
@@ -32,6 +33,9 @@ const GCSFUSE_PARENT_PROCESS_DIR = "gcsfuse-parent-process-dir"
 const Sequential = "Sequential"
 const Random = "Random"
 
+const MaxMiBsInUint64 uint64 = math.MaxUint64 >> 20
```

Comment:
I have seen this value is duplicated at two places (yaml_parser.go), can we keep at one place?

Reply:
I wanted the yaml_parser to have a constant MaxSupportedStatCacheMaxSizeInMb which would point to `util.MaxMiBsInUint64` but that was failing because of cycle dependency of package because of dependency of package util on logger, config. I have now removed those dependencies, by moving code around accordingly. Now, I have de-duped the calculation.